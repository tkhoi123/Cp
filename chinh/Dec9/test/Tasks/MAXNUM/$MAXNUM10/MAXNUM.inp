7
3145 777464 104791 314576 104 663340 678514 