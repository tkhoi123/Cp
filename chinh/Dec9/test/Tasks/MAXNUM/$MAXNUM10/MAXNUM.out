7774646785146633403145763145104791104